numpyro/contrib/einstein/stein_kernels.py

```diff
@@ -47,16 +45,15 @@ def compute(
         raise NotImplementedError
 
     def init(self, rng_key, particles_shape):
-        """
-        Initializes the kernel
-        :param rng_key: a JAX PRNGKey to initialize the kernel
-        :param tuple particles_shape: shape of the input `particles` in :meth:`compute`
-        """
+        """Initializes the kernel :param rng_key: a JAX PRNGKey to initialize the
+        kernel :param tuple particles_shape: shape of the input `particles` in
+        :meth:`compute`"""
         pass
 
 
 class RBFKernel(SteinKernel):
-    """Calculates the Gaussian RBF kernel function used in [1]. The kernel is given by
+    """Calculates the Gaussian RBF kernel function used in [1]. The kernel is given
+    by.
```

Comment:
no `dot` at the end here I think. Maybe move `The kernel is given by` to a new paragraph.